src/handle.rs

```diff
@@ -241,6 +277,51 @@ impl Handle {
         }
     }
 
+    pub fn set_bpf(self, bpf: Bpf) -> Result<Self, Error> {
+        let mut bpf = bpf;
```

Comment:
You can put the `mut` on the parameter you want:

```rust
    pub fn set_bpf(self, mut bpf: Bpf) -> Result<Self, Error> {
```

Reply:
Changed